array.test.js

```diff
@@ -0,0 +1,16 @@
+const analyze = require('./array');
+
+test('calculates average', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6]).average).toBe(4);
+});
+
+test('returns minimum number', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6]).min).toBe(1);
+});
+test('returns maximum number', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6]).max).toBe(8);
+});
+
+test('calculates length', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6]).length).toBe(6);
+});
```

Comment:
You could have a sinlge test for all of these:

```js
test('returns an object', () => {
  expect(analyze([1, 8, 3, 4, 2, 6])).toEqual({
    average: 4,
    min: 1,
    max: 8,
    length: 6,
  });
});
```
You could include some test to check what happens if an empty array is given, or if a value other than a number is given in the array.